hazelcast-sql/src/main/java/com/hazelcast/sql/impl/schema/MappingField.java

```diff
@@ -89,13 +89,21 @@ public MappingField setType(QueryDataType type) {
     }
 
     /**
-     * The external name of a field. For example, in case of IMap or Kafka,
+     * The external name of this field. For example, in case of IMap or Kafka,
```

Comment:
nit: in the same sentence we use "this" and "`this`" which might be slightly confusing